src/App.jsx

```diff
@@ -7,6 +7,10 @@ import Users from "./components/Users/Users";
 import Home from "./components/Home/Home";
 import { LoginContext } from "./contexts/LoginContext";
 import { useState } from "react";
+import Coding from "./components/Topics/Coding";
+import Cooking from "./components/Topics/Cooking";
```

Comment:
have a think about how you could make this far dry-er! They are all using the same api call, but have one different piece of information. It would be much nicer if you found a way to make one dynamic component